### ISSUE TITLE
Unnecessary emptiness check.

### ISSUE THREAD

**Issue by garyblankenship** [kalessil/phpinspectionsea#1173]
First, thank you for reporting a bug or making a request. That takes time and we appreciate that!

Let's start with common details:

Inspecting some legacy code.  An empty + OR statement with strlen() is a false report?

| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | Bug/False-Positive, Unnecessary emptiness check.                          |
| Plugin         | Php Inspections (EA Ultimate) w/subscription |
| Language level | PHP 7.1                      |

Current behaviour (description/screenshot:)
--
```
<?php
$invalid = empty($name) || strlen($name) > 10;
```
 
Seems to always be true when reached.

Expected behaviour
--
Should not report this false positive.

$name = null; // $invalid = true
$name = 0; // $invalid = true
$name = ""; // $invalid = true
$name = "1234567890123456789"; // $invalid = true
$name = "gary"; // $invalid = false

Environment details
---
Information from Help -> About dialog (contains IDE and it's version details).

Notes
---
- Bugs fixed with the highest priority, so features might be postponed. But it shouldn't stop you, right?
- The project support is powered by community funding via [Patreon](https://www.patreon.com/kalessil)

**Comment by kalessil** [comment]
That's quite strange behaviour... Thank you for reporting @garyblankenship, I'll take care of fixing this.

**Comment by kalessil** [comment]
Fixed!
